modules/sbt-coursier-shared/src/main/scala/coursier/sbtcoursiershared/InputsTasks.scala

```diff
@@ -147,55 +147,58 @@ object InputsTasks {
 
       val projectRefs = Structure.allRecursiveInterDependencies(state, projectRef)
 
-      val t = coursierProject.forAllProjectsOpt(state, projectRefs)
+      val t = coursierProject.forAllProjectsOpt(state, projectRefs :+ projectRef)
```

Comment:
I'm just following along here in https://github.com/sbt/sbt/pull/4925, but I am not sure why self projectRef is being added to as an inter-project dependency.